test/test_correctLik.jl

```diff
@@ -64,11 +65,29 @@ lik = logPseudoLik(d)
 end
 
 @testset "network estimation h=1" begin
-estNet = optTopRun1!(currT, 0.01,75, d,1, 1e-5,1e-6,1e-3,1e-4,
-                     false,true,Int[], 54, stdout,false,0.3)
-# topology, likAbs,Nfail, data,hmax, fRel,fAbs,xRel,xAbs,
-# verbose,closeN,numMoves, seed, logfile,writelog,probST,sout)
-@test loglik(estNet) < 0.00217
+  estNet = optTopRun1!(currT, 0.01,75, d,1, 1e-5,1e-6,1e-3,1e-4,
+                       false,true,Int[], 54, stdout,false,0.3,0.0)
+  # topology, likAbs,Nfail, data,hmax, fRel,fAbs,xRel,xAbs,
+  # verbose,closeN,numMoves, seed, logfile,writelog,probST,sout)
+  @test loglik(estNet) ≈ 0.110086447939308 || abs(loglik(estNet) - 0.002162891949473132) < 1e-8 # former is what RNG on my machine finds, latter is what GitHub actions finds
+end
+
+@testset "network estimation h=1 probQR=0.5" begin
+  Random.seed!(42)
+  estNet = optTopRun1!(currT, 0.01,75, d,1, 1e-5,1e-6,1e-3,1e-4,
+                       false,true,Int[], 54, stdout,false,0.3,0.5)
+  # topology, likAbs,Nfail, data,hmax, fRel,fAbs,xRel,xAbs,
+  # verbose,closeN,numMoves, seed, logfile,writelog,probST,sout)
+  @test loglik(estNet) ≈ 1.941159212747682e-7
+end
+
+@testset "network estimation h=1 probQR=0.75" begin
+  Random.seed!(42)
+  estNet = optTopRun1!(currT, 0.01,75, d,1, 1e-5,1e-6,1e-3,1e-4,
+                       false,true,Int[], 54, stdout,false,0.3,0.75)
+  # topology, likAbs,Nfail, data,hmax, fRel,fAbs,xRel,xAbs,
+  # verbose,closeN,numMoves, seed, logfile,writelog,probST,sout)
+  @test loglik(estNet) ≈ 0.110086447939308 || abs(loglik(estNet) - 0.002162891949473132) < 1e-8 # former is what RNG on my machine finds, latter is what GitHub actions finds
```

Comment:
Perhaps unrelated the the multithreading instability issue, but is this difference in RNG still present?

Reply:
I don't know. Can you test locally?